vaapi/vaapidisplay.cpp

```diff
@@ -320,4 +333,14 @@ DisplayPtr VaapiDisplay::create(const NativeDisplay& display)
 {
     return DisplayCache::getInstance()->createDisplay(display);
 }
+
+DisplayPtr VaapiDisplay::create(const NativeDisplay& display, VAProfile profile)
+{
+    SharedPtr<DisplayCache> cache = DisplayCache::getInstance();
+    const std::string name="hybrid";
```

Comment:
@last comment, nevermind I just realized you were probably talking about the VaapiV4L2Device in chromium...